skyportal/handlers/api/internal/profile.py

```diff
@@ -150,6 +152,16 @@ def patch(self):
             if len(username) < 5:
                 return self.error("Username must be at least five characters long.")
             user.username = username
+            if current_username != username:
+                user_group = (
+                    DBSession()
+                    .query(Group)
+                    .filter(Group.name == current_username)
+                    .first()
+                )
+                if user_group is not None:
+                    user_group.name = username
+                username_updated = True
```

Comment:
Should this be indented or not?

Reply:
@stefanv no, because we still use it to trigger a re-fetch of the users list